builtin/logical/database/path_config_connection.go

```diff
@@ -24,6 +25,8 @@ type DatabaseConfig struct {
 	// by each database type.
 	ConnectionDetails map[string]interface{} `json:"connection_details" structs:"connection_details" mapstructure:"connection_details"`
 	AllowedRoles      []string               `json:"allowed_roles" structs:"allowed_roles" mapstructure:"allowed_roles"`
+
+	RootCredentialsRotateStatements []string `json:"root_credentials_rotate_statements" structs:"credential_rotate_statements" mapstructure:"credential_rotate_statements"`
```

Comment:
Is there a reason for `structs` and `mapstructure` tags not having `root_credentials_rotate_statements`?